README.md

```diff
@@ -27,7 +27,7 @@ Support for **Mac OS** is not guaranteed (see [PR #61](https://github.com/nix-co
 ## Install
 
 ```
```

Comment:
usually is the syntax is set to `console`, it should make the line copy-pastable

Reply:
nevermind, console just does the highlighting